images/linux/scripts/installers/dotnetcore-sdk.sh

```diff
@@ -46,34 +46,35 @@ done
 sdks=()
 for version in ${versions[@]}; do
     release_url="https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/${version}/releases.json"
-    echo "${release_url}"
-    releases=$(curl "${release_url}")
+    download_with_retries "${release_url}" "." "${version}.json"
+    releases=$(cat "./${version}.json")
     sdks=("${sdks[@]}" $(echo "${releases}" | jq '.releases[]' | jq '.sdk.version'))
     sdks=("${sdks[@]}" $(echo "${releases}" | jq '.releases[]' | jq '.sdks[]?' | jq '.version'))
+    rm ./${version}.json
 done
 
 sortedSdks=$(echo ${sdks[@]} | tr ' ' '\n' | grep -v preview | grep -v rc | grep -v display | cut -d\" -f2 | sort -u -r)
+extract_dotnet_sdk() {
+    local ARCHIVE_NAME="$1"
+    set -e
+    dest="./tmp-$(basename -s .tar.gz $ARCHIVE_NAME)"; \
```

Comment:
Why do we need `; \` at end of lines?

Reply:
missed that, i'll remove this, thank you!